doc/base/classes.xml

```diff
@@ -5395,33 +5395,52 @@
 		</method>
 	</methods>
 	<members>
-		<member name="angular_damp" type="float" setter="set_angular_damp" getter="get_angular_damp" brief="">
+		<member name="angular_damp" type="float" setter="set_angular_damp" getter="get_angular_damp" brief="rate of spin loss">
+      The rate at which objects stop spinning in this area, excluding external forces' influences, measured as a fraction of speed lost per second (1.0 - stops immediately, 0.0 - never stops).
+			The period between physics calculations affects this rate (e.g. 1.0 - only stops movement after 1 second when physics is evaluated every 1 second). 
 		</member>
-		<member name="audio_bus_name" type="String" setter="set_audio_bus" getter="get_audio_bus" brief="">
+		<member name="audio_bus_name" type="String" setter="set_audio_bus" getter="get_audio_bus" brief="name of audio bus">
+      The name of the Area2D's audio bus.
 		</member>
-		<member name="audio_bus_override" type="bool" setter="set_audio_bus_override" getter="is_overriding_audio_bus" brief="">
+		<member name="audio_bus_override" type="bool" setter="set_audio_bus_override" getter="is_overriding_audio_bus" brief="Whether to override audio bus">
+      Whether to override the default audio bus with the Area2D's.
```

Comment:
Boolean members should follow styleguide: https://github.com/GDquest/godot-3-guides/blob/master/contribution-guides/class-reference-styleguide.md#use-if-true-to-describe-booleans

Reply:
I forgot to mention that I added all of these "if true" changes too.